CMakeLists.txt

```diff
@@ -11,14 +11,19 @@ set(OVXLIB_API_ATTR "__attribute__\(\(visibility\(\"default\"\)\)\)")
 message(${OVXLIB_API_ATTR})
 add_definitions(-DOVXLIB_API=${OVXLIB_API_ATTR})
 
-if("${CONFIG}" STREQUAL "A311D")
-    include(cmake/A311D.cmake)
-elseif("${CONFIG}" STREQUAL "S905D3")
-    include(cmake/S905D3.cmake)
-elseif("${CONFIG}" STREQUAL "vim3_android")
-    include(cmake/vim3_android.cmake)
+if(VIP_SDK_DIR AND EXISTS ${VIP_SDK_DIR})
```

Comment:
In bazel prebuilt-sdk/BUILD, we use external_viv_sdk to allow supply of external_sdk. Can you change VIP_SDK_DIR to match it? Not a big deal though.